man/fi_usnic.7.md

```diff
@@ -46,9 +43,26 @@ libfabric provider:
     MPI's *usnic* BTL.
   * *FI_EP_MSG* and *FI_EP_RDM* endpoints are implemented, but are
     only lightly tested.  It is likely that there are still some bugs
-    in these endpoint types.
-  * Other capabilities, such as the tag matching interface and RDMA
-    operations, are not yet implemented.
+    in these endpoint types.  RDMA is not yet supported.
+  * [`fi_provider`(7)](fi_provider.7.html) lists requirements for all
+    providers.  The following limitations exist in the *usnic*
+    provider:
+    * *fi_getname* is not supported on *FI_EP_DGRAM*, *FI_EP_MSG*, and
+      passive endpoints.
+    * multicast operations are not supported on *FI_EP_DGRAM* and
+      *FI_EP_RDM* endpoints.
+    * *FI_EP_MSG* endpoints only support connect, accept, and shutdown
+      operations.
+    * Passive endpoints only support listen operations.
+    * *FI_EP_DGRAM* endpoints support `fi_sendmsg()` and
+      `fi_recvmsg()`, but all flags are ignored.
+    * *FI_EP_RDM* and *FI_EP_MSG* endponts do not support
+      `fi_sendmsg()` and `fi_recvmsg()`.
+    * Address vectors only support `FI_AV_MAP`.
+    * `FI_SOCKADDR_IN6` address formats are not supported.
+    * No counters are supported.
+    * RDMA is not supported.
```

Comment:
Might be better said as "RMA" instead of "RDMA", to match up with the libfabric lingo for this.